src/bug-filing/common/markup/markup-factory.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export type MarkupFactory = {
```

Comment:
>Factory [](start = 18, length = 7)

I understand that you're using an abstract factory pattern, but I think "factory" is maybe not the most intuitive name for this; I think maybe `Formatter` expresses the responsibility more specifically

Reply:
Make sense. Updated.